simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java

```diff
@@ -156,9 +146,7 @@ NotifyReportRequest createNotifyReportRequest(@Nullable Integer requestId, boole
 
     private TransactionEventRequest createTransactionEvent(Integer evseId, Integer connectorId, TransactionEventRequest.TriggerReason reason, TransactionData transactionData,
                                                            TransactionEventRequest.EventType eventType, Instant currentDateTime, Long seqNo) {
-        List<MeterValue> meterValues = Collections.singletonList(new MeterValue().withTimestamp(ZonedDateTime.now(ZoneOffset.UTC)).withSampledValue(
-                Collections.singletonList(new SampledValue().withValue(eventType == STARTED ? ZERO : new BigDecimal("1010")))));
-        return createTransactionEvent(evseId, connectorId, reason, transactionData, eventType, currentDateTime, seqNo, meterValues);
+        return createTransactionEvent(evseId, connectorId, reason, transactionData, eventType, currentDateTime, seqNo, 1000);
```

Comment:
hardcoded value? I think we have a proper counter for consumption

Reply:
Yes but in some cases there is no values. It's better if i don't send anything then

Reply:
wait, but you do still send 1000 instead of nothing

Reply:
Yes, i will change the pr :)